scripts/h5_to_gif.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import h5py
+from PIL import Image
+
+def convert(file, speed=120):
+    """Convert images in an HDF5 file into a gif and save to cwd"""
+    # Getting images from HDF5 file
+    h5_file = h5py.File(file, 'r')
+    images = h5_file['entry']['data']['data']
+    # Converting to PIL.Image
+    images = [Image.fromarray(i).convert() for i in images]
+    # Saving as GIF
+    images[0].save(file.split('/')[-1].replace('.h5', '.gif'),
+                   save_all=True, append_images=images[1:],
+                   duration=speed, loop=0)
+    return
+
+
+if __name__ == "__main__":
+    # Arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--file', '-f', help='hdf5 file containing images',
+                        required=True)
+    parser.add_argument('--speed', '-s', type=int, nargs='?', default=120,
+                        help='playback speed for animated gifs, default 120')
```

Comment:
Is there a unit for this? How fast is 120?

Reply:
I have changed the name to duration and noted that the number corresponds to how long each frame of the gif lasts in milliseconds.